packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
             created
           example: 1587667174725
 
+    vod-asset:
+      type: object
+      table: vod_asset
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        userId:
+          type: string
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description:
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        duration:
+          type: number
+          description: Duration of the asset in seconds (float)
+          example: 23.8328
+        aspectRatio:
+          type: string
+          description: Aspect ratio of the asset
+          example: 16:9
```

Comment:
I wonder if we should keep the video-specific fields under a separate subobject here, WDYT? That so this same object could be later used for other file types, like for example images/thumbnails etc.

Reply:
I agree, I even think `aspectRatio` is redundant, as we have the `width` and `height` fields in both the track and profile objects?

Reply:
Yeah agreed! I think I'm good without that field as well.